exercises/high-scores/canonical-data.json

```diff
@@ -62,7 +62,7 @@
           "expected": [4]
         },
         {
-          "description": "Returns personal bests from a long list",
+          "description": "Returns three top scores from a long list",
```

Comment:
Should we also change the other tests that have "personal bests" in their descriptions? (Also maybe change phrasing to "top three scores"?)

Reply:
I didn't change those, because, in TDD manner, they can just return the `scores.sort.reverse` there. It's not until here that they have to `take(3)`.

Reply:
Ah, interesting. Ok 👍